### PR TITLE
.gitignore: new stuff

### PR DESCRIPTION
**Pull request by dzafman**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -61,3 +61,14 @@ GTAGS
...
+
+# Python specific
+/install-deps-python2.7
+/install-deps-python3
+
+# Ceph-disk
+/src/ceph-disk/
+
+# Other
+/src/pybind/mgr/dashboard_v2/
```
Comment (tchaikov) [inline .gitignore]: dashboard_v2 has been renamed. so we don't need this, do we?

Comment (dzafman) [reply .gitignore]: I was just basing this on my tree.  Any suggested improvements are welcome.

Comment (dzafman) [reply .gitignore]: I added those directories because they came up on "git status" as untracked.

**Comment by dzafman** [comment]
Dropping this for now

**Pull request closed without merging**